packages/workers-shared/package.json

```diff
@@ -46,6 +48,7 @@
 	},
 	"devDependencies": {
 		"@cloudflare/eslint-config-worker": "workspace:*",
+		"@cloudflare/vitest-pool-workers": "latest",
```

Comment:
How do y'all in ANT feel about us pulling from npm here rather using the workspace version? Circular dependency otherwise.

Reply:
Think this is good for now, we can always revisit if it becomes an issue